tools/codegen/model.py

```diff
@@ -1710,6 +1763,16 @@ def parse(op_name: str) -> 'OperatorName':
         assert str(r) == op_name, f'{str(r)} != {op_name}'
         return r
 
+    def with_base_name(self, base: str) -> 'OperatorName':
+        return OperatorName(
+            name=BaseOperatorName(
+                base=base,
+                inplace=self.name.inplace,
+                dunder_method=self.name.dunder_method,
+            ),
+            overload_name=self.overload_name
+        )
```

Comment:
also looks dead

Reply:
darn, i thought i found all of the dead code sites